### PR TITLE
Added Seive of Eratosthenes ,Issue #2488 fixed

### PR DESCRIPTION
**Pull request by anotherwebguy**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes.

### Description

Added the Seive of Eratosthenes method to find all the prime numbers within a range from 1 to n, given N
The algorithm finds all the numbers within the range which are prime.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#2488

### CODE REVIEW

File: C-Plus-Plus/other/sieve_Of_Eratosthenes.cpp [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+//printing the prime array
+void print_array(bool prime_array[] , int n)
+{
+	for (int i = 2; i <= n; i++)
+		if (!prime_array[i])
+			cout << i << " ";
+}
+
+
```
Comment (siddharth25pandey) [inline C-Plus-Plus/other/sieve_Of_Eratosthenes.cpp]: Remove the unnecessary blank lines

File: C-Plus-Plus/other/sieve_Of_Eratosthenes.cpp [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+	
+	cout << "Following are the prime numbers smaller "
+		<< " than or equal to " << n << endl;
+		
+	SieveOfEratosthenes(prime_array , n);
+	
+	return 0;
+}
+
+
```
Comment (siddharth25pandey) [inline C-Plus-Plus/other/sieve_Of_Eratosthenes.cpp]: Remove the unnecessary blank lines

File: C-Plus-Plus/other/sieve_Of_Eratosthenes.cpp [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+Input
+Enter N of range 1 to N : 45
+
+Output
+Following are the prime numbers smaller  than or equal to 45
+2 3 5 7 11 13 17 19 23 29 31 37 41 43
+
+*/
+
```
Comment (siddharth25pandey) [inline C-Plus-Plus/other/sieve_Of_Eratosthenes.cpp]: Remove the unnecessary blank lines

File: C-Plus-Plus/other/sieve_Of_Eratosthenes.cpp [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+Next cross out all the multiples of five except five itself. Continue this process until you have discovered as many prime numbers as you need.
+So in this by crossing out the non-prime numbers within a range 1 to N, we are only left with the prime numbers.
+
+*/
+#include <bits/stdc++.h>
+using namespace std;
+
+
+//printing the prime array
+void print_array(bool prime_array[] , int n)
```
Comment (siddharth25pandey) [inline C-Plus-Plus/other/sieve_Of_Eratosthenes.cpp]: Follow a good naming convention for variable n

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@anotherwebguy Do the modifications as suggested.
* Remove all the unnecessary blank lines
* Follow a good naming convention for variables.

**Comment by anotherwebguy** [comment]
done with the changes @siddharth25pandey

**Review by siddharth25pandey — APPROVED** [review]
@anotherwebguy Good to go, able to merge

**Comment by iamrajiv** [comment]
@anotherwebguy Sieve of Eratosthenes is already been added in #243.

**Comment by anotherwebguy** [comment]
> @anotherwebguy Sieve of Eratosthenes is already been added in #243.

Sorry my bad i didn't saw that so should i delete this pr?

**Pull request closed without merging**